README.md

```diff
@@ -0,0 +1,52 @@
+
+# Hello Microverse
+
+This is project is going to help us to master github flow and get used with linters for checking errors in our css ans Html files.
```

Comment:
- [ ] **OPTIONAL**
Please make sure that there are no grammar or misspelling mistakes in your written texts because
It's so important to make your repo looks professional, and it's mandatory in advanced (future) projects.
I highly recommend using [Grammarly](https://app.grammarly.com/) for that matter 😃